2016/ome-tiff-overview-potn/index.html

```diff
@@ -90,7 +40,10 @@
 
                 <!-- title page -->
                 <section>
-                    <h1>OME File Formats</h1>
+                    <h1>OME file formats</h1>
```

Comment:
Could do but everything looks a bit the same in `h2` and more. In my local testing, removing capitals was enough to prevent the word splitting (really!). Would be good to double check.